doc/source/conf.py

```diff
@@ -524,7 +524,8 @@ def add_line(self, line: str, source: str, *lineno: int) -> None:
     "pyspark": ("https://spark.apache.org/docs/latest/api/python/", None),
     "python": ("https://docs.python.org/3", None),
     "pytorch_lightning": ("https://lightning.ai/docs/pytorch/stable/", None),
-    "scipy": ("https://docs.scipy.org/doc/scipy/", None),
+    # TODO(can): undo this once scipy doc is back up
```

Comment:
nit: could you use your full github user id?

Reply:
updated \o/